c/extras/aggregator.cc

```diff
@@ -273,7 +276,7 @@ void Aggregator::group_2d_continuous(DataTablePtr& dt,
   auto c1 = static_cast<RealColumn<double>*>(dt->columns[1]);
   double* d_c0 = c0->elements_w();
   double* d_c1 = c1->elements_w();
```

Comment:
Hmm, if this method is not supposed to modify `dt`, then it should be declared `const`. Then `c0`, `c1`, `d_c0`, `d_c1` will be const too, and you wouldn't have to ask for a writable (`_w`) version of the data. Generally, you should request writable data only when you intend to write into it -- otherwise, you may be creating unnecessary copies.

Reply:
Oh, that's just something that is left from the old times. Thanks for catching it. I will also review the rest of the code if there is anything left like that.

Reply:
Fixed.